streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestClient.java

```diff
@@ -87,7 +87,7 @@ public void closeAsync() {
     }
 
     public void close() {
-        streams.close(Duration.ofSeconds(5));
+        streams.close(Duration.ofSeconds(60));
```

Comment:
The `SmokeTestIntegrationTest` fails with changes introduced in this PR and this is an attempt to fix it.